### PR TITLE
Capitalize first sentence of admonitions

### PR DESCRIPTION
**Pull request by Josh-Cena**
After the recent effort of admonition conversion, I discovered a few cases where the first sentence is not properly capitalized.

### CODE REVIEW

File: files/en-us/web/javascript/guide/using_promises/index.md [hunk 1]
```diff
@@ -204,8 +204,7 @@ Note how the code looks exactly like synchronous code, except for the `await` ke
 
 `async`/`await` builds on promises — for example, `doSomething()` is the same function as before, so there's minimal refactoring needed to change from promises to `async`/`await`. You can read more about the `async`/`await` syntax in the [async functions](/en-US/docs/Web/JavaScript/Reference/Statements/async_function) and [`await`](/en-US/docs/Web/JavaScript/Reference/Operators/await) references.
 
-> [!NOTE]
-> async/await has the same concurrency semantics as normal promise chains. `await` within one async function does not stop the entire program, only the parts that depend on its value, so other async jobs can still run while the `await` is pending.
+> **Note:** `async`/`await` has the same concurrency semantics as normal promise chains. `await` within one async function does not stop the entire program, only the parts that depend on its value, so other async jobs can still run while the `await` is pending.
```
Comment (estelle) [inline files/en-us/web/javascript/guide/using_promises/index.md:207]: here too.

File: files/en-us/web/http/proxy_servers_and_tunneling/proxy_auto-configuration_pac_file/index.md [hunk 2]
```diff
@@ -308,8 +308,7 @@ myIpAddress()
 
 Returns the server IP address of the machine Firefox is running on, as a string in the dot-separated integer format.
 
-> [!WARNING]
-> myIpAddress() returns the same IP address as the server address returned by **`nslookup localhost`** on a Linux machine. It does not return the public IP address.
+> **Warning:** `myIpAddress()` returns the same IP address as the server address returned by **`nslookup localhost`** on a Linux machine. It does not return the public IP address.
```
Comment (estelle) [inline files/en-us/web/http/proxy_servers_and_tunneling/proxy_auto-configuration_pac_file/index.md:311]: is a revert for this one intentional?

**Review by estelle — APPROVED** [review]
just two questions about whether a revert to `**Note**` is intentional.

File: files/en-us/web/http/proxy_servers_and_tunneling/proxy_auto-configuration_pac_file/index.md (hunk 2 above)
Comment (OnkarRuikar) [reply files/en-us/web/http/proxy_servers_and_tunneling/proxy_auto-configuration_pac_file/index.md:311]: We could've reworded it like ```The `myIpAddress()` function returns...```

Comment (Josh-Cena) [reply files/en-us/web/http/proxy_servers_and_tunneling/proxy_auto-configuration_pac_file/index.md:311]: @OnkarRuikar sure, but we must not let writers rephrase what they have already written just to please a buggy formatter.

**Pull request merged**